src/client/app/modules/LoaderSpinner.js

```diff
@@ -0,0 +1,21 @@
+(() => {
+  class LoaderSpinner {
+    constructor() {
+      this.div = $(".cover");
```

Comment:
Can you please use a more descriptive name for this element? Also please use [this naming convention](https://stackoverflow.com/questions/205853/why-would-a-javascript-variable-start-with-a-dollar-sign) to indicate that there is a Jquery element selected